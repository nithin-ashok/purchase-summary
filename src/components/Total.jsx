import React from "react";


const Total = ({total}) => (
    <table className="table">
        <tbody>
        <tr className="font-bold">
            <td className="font-large">
                Est. total
            </td>
            <td className="text-right font-xlarge">
                {`$${total}`}
            </td>
        </tr>
        </tbody>
    </table>
);
export default Total;