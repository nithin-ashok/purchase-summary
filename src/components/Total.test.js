import React from 'react';
import Total from "./Total";
import renderer from 'react-test-renderer';

it("renderes correctly Total component", () => {
    let total = 100;
    const tree = renderer.
    create(<Total total={total} />).toJSON();

    expect(tree).toMatchSnapshot();
});