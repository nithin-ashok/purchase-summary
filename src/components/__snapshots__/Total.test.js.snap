// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderes correctly Total component 1`] = `
<table
  className="table"
>
  <tbody>
    <tr
      className="font-bold"
    >
      <td
        className="font-large"
      >
        Est. total
      </td>
      <td
        className="text-right font-xlarge"
      >
        $100
      </td>
    </tr>
  </tbody>
</table>
`;
